src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -194,8 +179,7 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConformBidState
         status,
         message_header: messageHeader,
         message_description_md: messageDescriptionMd,
-        winning,
-        bid: this.props.bid,
+        winning: status === "WINNING",
```

Comment:
Ideally, we should pass the entire `BidderPositionResult` to the `<BidResult>` component and let it display an appropriate message, but this should be good for now.